pallets/issue/src/benchmarking.rs

```diff
@@ -64,6 +64,8 @@ benchmarks! {
 	request_issue {
 		let origin: T::AccountId = account("Origin", 0, 0);
 		let vault_id = get_vault_id::<T>();
+		// this amount is when running the benchmark with the testchain
+		// let amount = 1000_0000u32.into();
```

Comment:
Why this comment? I need to change the amount when running benchmarks with the testchain but use the `1000` otherwise? Can we change it such that it always uses `100_00000u32` then so that it works in both cases?

Reply:
changed amount to the fixed `1000_0000u32.into()`